cli/src/config.rs

```diff
@@ -210,7 +210,7 @@ pub struct AnnotatedValue {
 /// Sources from the lowest precedence:
 /// 1. Default
 /// 2. Base environment variables
-/// 3. [User config](https://github.com/martinvonz/jj/blob/main/docs/config.md#configuration)
+/// 3. [User config](https://martinvonz.github.io/jj/latest/config/#configuration)
```

Comment:
The `#configuration` can be removed here, it does no good. (I randomly tried this link)

Reply:
Removed `#configuration`.